### ISSUE TITLE
Bad resource path (css, js) - build issue

### ISSUE THREAD

**Issue by motorina0** [codenvy/codenvy#711]
The onprem `allinone` zip built locally does not show login page.
I'm describing below one of the problems and the found solution.
Please close this issues if the solution is accepted.

**Reproduction Steps:**
- run `mvn clean install`
- copy  `onpremises-ide-packaging-tomcat-codenvy-allinone-4.6.2.zip` to `/etc/puppet/files/servers/prod/aio`
- run `puppet agent -t`
- open `http://codenvy.onprem/`

**Expected behavior:**
The login page is open

**Observed behavior:**
A blank page opens. The username/password fields are not displayed.

**Codenvy version:** 4.62
**OS and version:** Windows 7

**Additional information:**

Problem only started happening recently, didn't happen in an older version of Codenvy: [No]
-only happen when I've deployed my own `allinone` zip. It worked fine with the default installation (by puppet)
Problem can be reliably reproduced: [Yes]

Issue:
- go to `allinone`zip and open `\webapps\site\login.html`
  - the following differences can be seen from the original version:
    ![login_html_new_vs_old](https://cloud.githubusercontent.com/assets/2951406/18378349/21a368e0-7675-11e6-9ff8-fd52b06b129c.png)
  - that is, the new `login.html` file does not have the generated `id` as suffix for some `css`and `js` resources
- check `/site/target/dist/prod/site/styles`. These files are present (only with suffix).
  - bootstrap-e5659bcf.css
  - style-b404a08c.css
- check `/site/target/dist/prod/site/scripts`:
  - amd-app-6c7a32ec.js

**Workaround**
- also copy the files with clean name (without suffix)
- go to `/site/gulpfile.js`
- replace 

```
gulp.task('copy_prod' ...
    paths.prod+'**/amd-app-*.js', // minified JS
    paths.prod+'**/*-*.css', // minified CSS

gulp.task('copy_onprem_se', ...
    paths.onpremSE+'**/amd-app-*.js', // minified JS
    paths.onpremSE+'**/*-*.css', // minified CSS
```

  with

```
gulp.task('copy_prod' ...
    paths.prod+'**/amd-app*.js', // minified JS
    paths.prod+'**/*.css', // minified CSS

gulp.task('copy_onprem_se', ...
    paths.onpremSE+'**/amd-app*.js', // minified JS
    paths.onpremSE+'**/*.css', // minified CSS
```
- basically remove the dash ("-")

This works, but it is just an workaround. Why didn't it worked out of the box?

**Comment by ddementieva** [comment]
@motorina0 If you think it's a proper fix, create a pull request and our engineers will review.

**Comment by TylerJewell** [comment]
@motorina0 - I like this fix. Can you please open a PR, ensure you have the Eclipse CLA executed, and we'll get it included.

**Comment by motorina0** [comment]
PR: https://github.com/codenvy/codenvy/pull/1113

**Comment by vkuznyetsov** [comment]
In 5.3.0 can't reproduce this issue.
